src/schema/rules/files/raw/channels.yaml

```diff
@@ -8,22 +8,15 @@ channels:
   datatypes:
     - eeg
     - ieeg
+    - meg
     - nirs
   entities:
     subject: required
     session: optional
     task: required
     acquisition: optional
-    run: optional
-
-# MEG has an additional entity available
-channels__meg:
-  $ref: rules.files.raw.channels.channels
-  datatypes:
-    - meg
-  entities:
-    $ref: rules.files.raw.channels.channels.entities
     processing: optional
```

Comment:
As I understand it, you propose permitting `_proc-` for channels.tsv in `eeg/ieeg/nirs`, but not for EEG/iEEG/NIRS data files?

Reply:
I would be happy  to add _proc to those (and more if feasible) modalities since IMHO it is generic enough.  Added in 16bade398de9b03e6fd8247f8a5d9cabde71d354

Reply:
channels are not a modality nor a measurement, but only the metadata that complements the eeg/ieeg/meg data+json file. However, if the data file itself is "processed", I would be happy to see the corresponding `channels.tsv` file name still matching the data file. E.g. in https://github.com/bids-standard/bids-examples/tree/master/ds000117/derivatives/meg_derivatives/sub-01/ses-meg/meg there is no `channels.tsv` file (nor an `events.tsv` file for that matter). In that specific example, I think it makes sense if `channels.tsv` would have `proc-sss` as part of the file name.

Reply:
good, so we agree to agree if I got it right ;)! I will mark this discussion resolved to save some screen space. Please unresolve if I misunderstood